config/rbac/kustomization.yaml

```diff
@@ -12,4 +12,10 @@ resources:
 # - metrics_auth_role_binding.yaml
 # - metrics_reader_role.yaml
 apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
\ No newline at end of file
+kind: Kustomization# For each CRD, "Editor" and "Viewer" roles are scaffolded by
+# default, aiding admins in cluster management. Those roles are
```

Comment:
This can be removed as we don't ship the CRD in the operator

Reply:
Done